docs/source/markdown/podman-build.1.md

```diff
@@ -845,23 +845,23 @@ $ podman build .
 
 $ podman build -f Containerfile.simple .
 
-$ cat $HOME/Dockerfile | podman build -f - .
+$ cat $HOME/Containerfile | podman build -f - .
 
-$ podman build -f Dockerfile.simple -f Containerfile.notsosimple .
+$ podman build -f Containerfile.simple -f Containerfile.notsosimple .
 
-$ podman build -f Dockerfile.in $HOME
+$ podman build -f Containerfile.in $HOME
 
 $ podman build -t imageName .
 
-$ podman build --tls-verify=true -t imageName -f Dockerfile.simple .
+$ podman build --tls-verify=true -t imageName -f Containrfile.simple .
```

Comment:
Still typo :-(

Reply:
Yes, let's please fix this before merge.